### PR TITLE
EMAIL_USE_TLS/EMAIL_USE_SSL are mutually exclusive, so only set one o…

### LINKED ISSUE #624: [BUG] mailman-web: `EMAIL_USE_TLS/EMAIL_USE_SSL`
**Tag/version of Container Images**
`latest` as of today ([34b5c3af84cc](https://hub.docker.com/layers/maxking/mailman-core/latest/images/sha256-34b5c3af84cce9b3d3a758e12cb4edeabbc1559b4fba14cb85532471b5a0f199?context=explore))

My mail server does support both SSL/TLS & STARTTLS flawlessly in all sorts of configurations.

### To Reproduce

```yaml
services:
  web:
    image: maxking/mailman-web
    environment:
      - SMTP_HOST=<mail.example.com>
      - SMTP_PORT=465
      - SMTP_HOST_USER=<user>
      - SMTP_HOST_PASSWORD=<pass>
      - SMTP_USE_SSL=True
      - SMTP_USE_TLS=False
```

`$ docker-compose exec mailman-web /opt/mailman-web/manage.py sendtestemail hostmaster@example.com`

**Hangs**. 

```yaml
services:
  web:
    image: maxking/mailman-web
    environment:
      - SMTP_HOST=<mail.example.com>
      - SMTP_PORT=587
      - SMTP_HOST_USER=<user>
      - SMTP_HOST_PASSWORD=<pass>
      - SMTP_USE_TLS=True
      - SMTP_USE_SSL=False
```

`$ docker-compose exec mailman-web /opt/mailman-web/manage.py sendtestemail hostmaster@example.com`

**Works**.

### Logs

I couldn't find any error message in any of the logs in `/opt/mailman-web-data/logs` pertaining to SMTP, as such I have no more information.

### PR DESCRIPTION
**Pull request by raph-topo**
Fix #624 like this?

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ change them unless you know what you want.
 - `SMTP_HOST_PASSWORD`: Default is an empty string.
 
 - `SMTP_USE_TLS`: Specifies wheather the SMTP connection is encrypted
-  via TLS. Default is `False`.
+  via TLS. Default is `False`. (EMAIL_USE_TLS/EMAIL_USE_SSL are mutually exclusive, so only set one of those settings.)
```
Comment (maxking) [inline web/README.md:59]:
```suggestion
  via TLS. Default is `False`. (`EMAIL_USE_TLS`/`EMAIL_USE_SSL` are mutually exclusive, so only set one of those settings.)
```

File: web/README.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ change them unless you know what you want.
...
 
 - `SMTP_USE_SSL`: Specifies wheather the SMTP connection is encrypted
-  via SSL. Default is `False`.
+  via SSL. Default is `False`. (EMAIL_USE_TLS/EMAIL_USE_SSL are mutually exclusive, so only set one of those settings.)
```
Comment (maxking) [inline web/README.md:62]:
```suggestion
  via SSL. Default is `False`. (`EMAIL_USE_TLS`/`EMAIL_USE_SSL` are mutually exclusive, so only set one of those settings.)
```

**Review by maxking — COMMENTED** [review]
Use backticks for variables.

**Comment by maxking** [comment]
Overall, lgtm! Added backticks to the variables.

**Comment by raph-topo** [comment]
Thanks! (I copy-pasted from the Django error message as is.)

**Pull request merged**